src/style.css

```diff
@@ -0,0 +1,120 @@
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+.bigForm {
+  position: relative;
+  margin-top: 20px;
+}
+
+.iconBig {
+  display: block;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.bigOne {
+  height: 70px;
+  width: 100%;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigOne:hover {
+  background: #fff;
+  border: 1px solid #f6f6f7;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigOne:focus {
+  font-family: "Avenir", sans-serif;
+  font-weight: 800;
+  box-sizing: border-box;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #f6f6f7;
+  border-radius: 3px;
+  outline: none;
+}
+
+.bigOne::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.smallOne {
+  height: 42px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

Comment:
Now you are repeating a lot of styles,  use this hind from checklist pls
![image](https://user-images.githubusercontent.com/80700247/229276128-ed3a283b-2af4-46b1-ad8d-15407eed4ce9.png)

Apply it everywhere